### PR TITLE
Add transfer-ledger-create

### PR DESCRIPTION
**Pull request by ApolloUnicorn**
Enables signing of txns using Ledger held private key.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM nginx:1.17.5
 
-ENV GIT_URL https://github.com/kadena-community/kadena-transfer-js
+ENV GIT_URL https://github.com/obsidiansystems/kadena-transfer-js.git
```
Comment (jmininger) [inline Dockerfile:3]: Can we change this back

File: docs/transfer-ledger-create.html [hunk 2]
```diff
@@ -0,0 +1,1092 @@
...
+         if (document.getElementById("receiver-exist").hidden === false){
+           keyset = JSON.parse(keyset)
+           if (validateKeyset(keyset)) return true;
+           else if (!!keyset.keysetref) return true;
+           else return false;
+         } else return true;
+       }
+       
+       $.fn.form.settings.rules.ledgerErrorPK = function(sig) {
+          //TO DO figure out a good rule for that
```
Comment (jmininger) [inline docs/transfer-ledger-create.html:94]: Does this need to be addressed or can it be taken out?

File: docs/transfer-ledger-create.html [hunk 3]
```diff
@@ -0,0 +1,1092 @@
...
+             method: "POST",
+             body: JSON.stringify(cmd)
+         };
+     };
+
+     // INITIATION FUNCTIONS
+     window.addEventListener('load', function (event) {
+       document.getElementById("getSig-button").addEventListener("click", async function(e){
+            dimBody("Waiting for ledger signature");
+            await signWithLedger();
```
Comment (jmininger) [inline docs/transfer-ledger-create.html:246]: When i reject on the ledger i see an unhandled exception in the browser console. The dimmer stays on and im unable to try again without reloading the whole page. Maybe we could wrap this in a `try` and hide the dimmer on any exceptions as well?

File: docs/transfer-ledger-create.html [hunk 4]
```diff
@@ -0,0 +1,1092 @@
...
+            $("body").dimmer("hide");
+       })
+
+       document.getElementById("getPubkey-button").addEventListener("click", async function(e){
+            dimBody("Waiting for ledger public key");
+            await getPublicKeyFromLedger();
+            $("body").dimmer("hide");
```
Comment (jmininger) [inline docs/transfer-ledger-create.html:253]: Same here

**Review by jmininger — CHANGES_REQUESTED** [review]

File: docs/transfer-ledger-create.html (hunk 3 above)
Comment (ApolloUnicorn) [reply docs/transfer-ledger-create.html:246]: fixed in https://github.com/obsidiansystems/kadena-transfer-js/pull/5

File: docs/transfer-ledger-create.html (hunk 4 above)
Comment (ApolloUnicorn) [reply docs/transfer-ledger-create.html:253]: https://github.com/obsidiansystems/kadena-transfer-js/pull/5

File: Dockerfile (hunk 1 above)
Comment (ApolloUnicorn) [reply Dockerfile:3]: https://github.com/obsidiansystems/kadena-transfer-js/pull/5

File: docs/transfer-ledger-create.html (hunk 2 above)
Comment (ApolloUnicorn) [reply docs/transfer-ledger-create.html]: removed/unused

**Pull request merged**